weave/trace_server/opentelemetry/constants.py

```diff
@@ -0,0 +1,121 @@
+
+This would be the resulting dict dumped to clickhouse:
+{ "prompt_tokens": 30, "completion_tokens": 40, "total_tokens": 70 }
+"""
+
+# These mappings prioritize standards in a specific order for each attribute type.
+
+# INPUT_KEYS: Maps attribute keys that represent user prompts or inputs to LLMs
+# Priority is given to standards in this order:
+# This is used to populate the `inputs_dump` column in clickhouse
+from weave.trace_server.opentelemetry.helpers import try_parse_int
+
+INPUT_KEYS = [
+    "input.value",  # From OpenInference standard - highest priority
+    "gen_ai.prompt",  # From OpenTelemetry AI semantic conventions
+    "mlflow.spanInputs",  # From MLFlow's tracking format
+    "traceloop.entity.input"  # From Traceloop's conventions
+    "input",  # Generic fallback for Pydantic models - lowest priority
+]
+
+# OUTPUT_KEYS: Maps attribute keys that represent model completions or outputs
+# Priority is given to standards in this order:
+# This is used to populate the `output_dump` column in clickhouse
+OUTPUT_KEYS = [
+    "output.value",  # From OpenInference standard - highest priority
+    "gen_ai.completion",  # From OpenTelemetry AI semantic conventions
+    "mlflow.spanOutputs",  # From MLFlow's tracking format
+    "gen_ai.content.completion",  # From OpenLit project's format
+    "traceloop.entity.output",  # From Traceloop's conventions
+    "output",  # Generic fallback for Pydantic models - lowest priority
+]
+
+# USAGE_KEYS: Maps internal Weave usage metric names to their equivalent keys in
+# various telemetry standards. Used for token counting and usage statistics.
+# Used to populate the usage field of `summary_dump` in clickhouse
+# The tuples represent handlers which the value should be passed through when found
+# Never assume that the value is of a certain type or error, conventions provide no guarantees
+USAGE_KEYS = {
+    # Maps Weave's "prompt_tokens" to keys from different standards
+    "prompt_tokens": [
+        ("gen_ai.usage.prompt_tokens", try_parse_int),
+        ("llm.token_count.prompt", try_parse_int),
+    ],
+    # Maps Weave's "completion_tokens" to keys from different standards
+    "completion_tokens": [
+        ("gen_ai.usage.completion_tokens", try_parse_int),
+        ("llm.token_count.completion", try_parse_int),
+    ],
+    # Maps Weave's "total_tokens" to keys from different standards
+    "total_tokens": [
+        ("llm.usage.total_tokens", try_parse_int),
+        ("llm.token_count.total", try_parse_int),
+    ],
+}
+
+# ATTRIBUTE_KEYS: Maps common LLM call metadata attributes to the types of attributes expected in weave traces
+# This is used to populate the `attributes_dump` column in clickhouse
+# Prior to dumping, the full span is dumped under another key not listed here called `otel_span`
+ATTRIBUTE_KEYS = {
+    # System prompt/instructions
+    "system": [
+        "gen_ai.system",  # OpenTelemetry AI
+        "llm.system",  # OpenInference
```

Comment:
Likely not a big deal, but in our Input / Output keys OpenInference is before OT AI

Reply:
I'll switch it real quick for consistency but it shouldn't actually make a difference here, the same information is recorded in each key so even if they have both it will work properly.